### PR TITLE
Improve debug testing

### PR DESCRIPTION
**Pull request by lecourtoisflo**
Also update tests references, as majority has the same results (see #76 )

### CODE REVIEW

File: sources/Outputs/src/Job.cpp [hunk 1]
```diff
@@ -130,5 +132,106 @@ Job::writeOutputs() {
...
+  attrs.clear();
+  formatter->endElement();  // simulation
+
+  // outputs
+  auto outputs = jobEntry->getOutputsEntry();
+  attrs.add("directory", outputs->getOutputsDirectory());
+  formatter->startElement("dyn", "outputs", attrs);
+  attrs.clear();
+
+  auto logs = outputs->getLogsEntry();
```
Comment (rosiereflo) [inline sources/Outputs/src/Job.cpp:214]: I think the line to generate the final IIDM is missing no?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: sources/Outputs/src/Job.cpp (hunk 1 above)
Comment (lecourtoisflo) [reply sources/Outputs/src/Job.cpp:214]: No, it is done automattically by the formatter

**Review by rosiereflo — APPROVED** [review]

**Review by PerninGuillaume — COMMENTED** [review]
Perhaps this exportJob function should be tested

**Comment by rosiereflo** [comment]
In my humble opinion it is not critical to test a debug method

**Review by PerninGuillaume — APPROVED** [review]

**Pull request merged**